package-overrides/github/jamesflorentino/nanoscrollerjs@0.8.4.json

```diff
@@ -0,0 +1,13 @@
+{
+    "main": "javascripts/jquery.nanoscroller.js",
+    "format": "global",
+    "directories": {
+      "lib": "bin"
+    },
+    "shim": {
+      "javascripts/jquery.nanoscroller": [
+        "jquery",
```

Comment:
Both `jquery` and `css` need to be added into the `dependencies` declaration so they are downloaded on install.